include.js

```diff
@@ -2243,12 +2243,16 @@ let cards = [
     githubLink: 'https://github.com/PentesterPriyanshu'
   },
   {
-    artName: 'Solar System',
-    pageLink: './Art/Shubhamkashyap1601/solar.html',
-    imageLink: './Art/Shubhamkashyap1601/solar.gif',
-    author: 'Shubham Kashyap',
-    githubLink: 'https://github.com/Shubhamkashyap1601'
```

Comment:
Please leave the entries from other students. You are currently deleting the entry.

Reply:
Please check now. I did it by mistake.